plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -132,6 +132,16 @@ limitations under the License.
                </xs:documentation>
             </xs:annotation>
          </xs:attribute>
+         <xs:attribute name="includesAllDependencies" type="xs:boolean" default="true">
```

Comment:
An idle thought, but would it be worth making this a `dependencies="ignore"` and `"include"`?  Are there other possibilities for handling dependencies like _only-immediate_ dependencies?  Or including _optional_ dependencies too?

Reply:
Wouldn't surprise me. The Java classes will diverge a little further from the XML but that's fine. Let me play with that.